File: todo-v1-app-android/app/src/main/java/com/example/todo/Models/TodoModel.java
package com.example.todo.Models;

public class TodoModel{

    public int id;
    public String task;
    public boolean is_completed;

    public TodoModel(int i, String s, boolean b){
        this.id = i;
        this.task = s;
        this.is_completed = b;
    }

}

File: todo-v1-app-ios/todo/Models/TodoModel.swift
//
//  Todo.swift
//  todo
//
//  Created by Atharva Parkhe on 12/02/22.
//

import Foundation

struct Todo : Codable{
    let id: Int
    let task: String
    let is_completed: Bool
}

struct PostTodo: Codable {
    let task : String
}

struct PatchTodo: Codable {
    let is_completed : Bool
}
